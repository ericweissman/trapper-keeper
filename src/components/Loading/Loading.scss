.loading {
  font-family: "Lato", sans-serif;
  font-size: 20px;
  color: #005b4f;
  padding-top: 50px;
  text-align: center
}