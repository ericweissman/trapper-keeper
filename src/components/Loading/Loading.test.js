import React from 'react'
import Loading from './Loading'
import { shallow } from 'enzyme'

describe('Loading', () => {
  let wrapper
  it('should have the correct snapshot', () => {
    wrapper = shallow(<Loading />)
    expect(wrapper).toMatchSnapshot()
  })
})