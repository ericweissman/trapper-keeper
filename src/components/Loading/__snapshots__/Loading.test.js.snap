// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should have the correct snapshot 1`] = `
<h3
  className="loading"
>
  Loading...
</h3>
`;
